application/holder/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "chipmunk",
-    "version": "3.4.24",
+    "version": "3.5.0",
```

Comment:
let's stick to semantic versioning: increase last digit if no API was changed and if no major new features are introduced.
If you take a look at rust crates, they often have hight 2-digits in the last version number (e.g.1.0.78)

Reply:
I've shifted major part because `breadcrumbs` looks like a new feature, but not patch